cmd/test/test.go

```diff
@@ -165,60 +165,74 @@ func newTestCommand() *cobra.Command {
 			}
 
 			if !runner.Quiet || exitCode != 0 {
-				if runner.Output == OutputAppstudio {
-					// The appstudio format is unknown to Conftest so we handle it ourselves
+				for _, outputAndFormat := range outputFormats {
+					parts := strings.SplitN(outputAndFormat, "=", 2)
 
-					if resultsErr != nil {
-						return appstudioErrorHandler(runner.NoFail, "running test", resultsErr)
+					format := parts[0]
+					if format == "" {
+						// default to stdout
+						format = output.OutputStandard
 					}
 
-					report := appstudioReport(results, runner.Namespace)
-					reportOutput, err := json.Marshal(report)
-					if err != nil {
-						return appstudioErrorHandler(runner.NoFail, "output results", err)
+					outputFilePath := ""
+					if len(parts) == 2 {
+						outputFilePath = parts[1]
 					}
+					if format == OutputAppstudio {
+						// The appstudio format is unknown to Conftest so we handle it ourselves
 
-					if outputFilePath != "" {
-						err := os.WriteFile(outputFilePath, reportOutput, 0600)
+						if resultsErr != nil {
+							return appstudioErrorHandler(runner.NoFail, "running test", resultsErr)
+						}
+
+						report := appstudioReport(results, runner.Namespace)
+						reportOutput, err := json.Marshal(report)
 						if err != nil {
-							return fmt.Errorf("creating output file: %w", err)
+							return appstudioErrorHandler(runner.NoFail, "output results", err)
 						}
-					}
-					fmt.Fprintln(cmd.OutOrStdout(), string(reportOutput)+"\n")
 
-				} else {
-					// Conftest handles the output
+						if outputFilePath != "" {
+							err := os.WriteFile(outputFilePath, reportOutput, 0600)
+							if err != nil {
+								return fmt.Errorf("creating output file: %w", err)
+							}
+						}
+						fmt.Fprintln(cmd.OutOrStdout(), string(reportOutput)+"\n")
```

Comment:
Maybe this would do it:

```suggestion
						else {
						    fmt.Fprintln(cmd.OutOrStdout(), string(reportOutput)+"\n")
						}
```